src/v8runtime/V8Inspector.h

```diff
@@ -9,8 +9,14 @@
 
 #include <cxxreact/MessageQueueThread.h>
 #include <condition_variable>
-#include "jsinspector/InspectorInterfaces.h"
 #include "v8-inspector.h"
+#if REACT_NATIVE_TARGET_VERSION >= 73
+#include "jsinspector-modern/InspectorInterfaces.h"
+namespace react = facebook::react::jsinspector_modern;
```

Comment:
this feels not quite safe to alias jsinspector_modern as react namespace.
how about just use jsinspector?

```diff
diff --git a/node_modules/react-native-v8/src/v8runtime/V8Inspector.cpp b/node_modules/react-native-v8/src/v8runtime/V8Inspector.cpp
index 2976039..7a1fb13 100644
--- a/node_modules/react-native-v8/src/v8runtime/V8Inspector.cpp
+++ b/node_modules/react-native-v8/src/v8runtime/V8Inspector.cpp
@@ -15,8 +15,6 @@
 #include "folly/json.h"
 #include "v8.h"
 
-namespace react = facebook::react;
-
 namespace rnv8 {
 
 static const char kInspectorName[] = "React Native V8 Inspector";
@@ -100,7 +98,7 @@ void InspectorFrontend::sendNotification(
   client_->SendRemoteMessage(message->string());
 }
 
-class LocalConnection : public react::ILocalConnection {
+class LocalConnection : public jsinspector::ILocalConnection {
  public:
   LocalConnection(std::weak_ptr<InspectorClient> weakClient)
       : weakClient_(weakClient) {}
@@ -193,11 +191,11 @@ void InspectorClient::ConnectToReactFrontend() {
   std::lock_guard<std::mutex> lock(connectionMutex_);
 
   auto weakClient = CreateWeakPtr();
-  pageId_ = react::getInspectorInstance().addPage(
+  pageId_ = jsinspector::getInspectorInstance().addPage(
       inspectorName_,
       "V8",
       [weakClient = std::move(weakClient)](
-          std::unique_ptr<react::IRemoteConnection> remoteConn) {
+          std::unique_ptr<jsinspector::IRemoteConnection> remoteConn) {
         auto client = weakClient.lock();
         if (client) {
           client->remoteConn_ = std::move(remoteConn);
@@ -212,7 +210,7 @@ void InspectorClient::Disconnect() {
   if (remoteConn_) {
     remoteConn_->onDisconnect();
   }
-  react::getInspectorInstance().removePage(pageId_);
+  jsinspector::getInspectorInstance().removePage(pageId_);
 }
 
 void InspectorClient::DisconnectFromReactFrontend() {
diff --git a/node_modules/react-native-v8/src/v8runtime/V8Inspector.h b/node_modules/react-native-v8/src/v8runtime/V8Inspector.h
index 7eeb2f1..15606f3 100644
--- a/node_modules/react-native-v8/src/v8runtime/V8Inspector.h
+++ b/node_modules/react-native-v8/src/v8runtime/V8Inspector.h
@@ -9,8 +9,14 @@
 
 #include <cxxreact/MessageQueueThread.h>
 #include <condition_variable>
-#include "jsinspector/InspectorInterfaces.h"
 #include "v8-inspector.h"
+#if REACT_NATIVE_TARGET_VERSION >= 73
+#include "jsinspector-modern/InspectorInterfaces.h"
+namespace jsinspector = facebook::react::jsinspector_modern;
+#else
+#include "jsinspector/InspectorInterfaces.h"
+namespace jsinspector = facebook::react;
+#endif
 
 namespace rnv8 {
 
@@ -84,7 +90,7 @@ class InspectorClient final
   v8::Isolate *isolate_;
   v8::Global<v8::Context> context_;
 
-  std::unique_ptr<facebook::react::IRemoteConnection> remoteConn_;
+  std::unique_ptr<jsinspector::IRemoteConnection> remoteConn_;
   std::string inspectorName_;
 
   std::mutex connectionMutex_;
```

Reply:
I agree with you, I've already made the change from "react" to "jsinspector".